Resources/input/fsh/ISiKKalender.fsh

```diff
@@ -20,10 +20,19 @@ Id: ISiKKalender
 * actor[Akteur] only Reference(Practitioner or HealthcareService)
 * actor[Akteur].reference 1..1 MS
 * extension MS
-* extension contains ISiKKalenderNameExtension named Name 0..1 MS
+* extension contains http://hl7.org/fhir/5.0/StructureDefinition/extension-Schedule.name named KalenderName 0..1 MS
 
-Extension: ISiKKalenderNameExtension
-Id: ISiKKalenderNameExtension
+// das muss weg
+//Extension: ISiKKalenderNameExtension
+//Id: ISiKKalenderNameExtension
+//* value[x] only string
+//* valueString 1..1 MS
+
+// This extension can be safely removed as soon as a package for R5 backport extensions is published and referenced by this project
+//WIP
+Extension: ScheduleName
+Id: ScheduleName
+* ^url = "http://hl7.org/fhir/5.0/StructureDefinition/extension-Schedule.name"
 * value[x] only string
 * valueString 1..1 MS
```

Comment:
Sollte 0..1 sein, da es in R5 auch ein optionales Element ist.

Reply:
... und MS removed, richtig?